cerberus/kubernetes/client.py

```diff
@@ -109,3 +109,30 @@ def monitor_component(iteration, component_namespace):
     logging.info("Iteration %s: %s: %s"
                  % (iteration, component_namespace, watch_component_status))
     return watch_component_status, failed_component_pods
+
+
+def monitor_cluster_operator(iteration, cluster_operators):
+
+    failed_operators = []
+    for operator in cluster_operators['items']:
+
+        #loop through the conditions in the status section to find the dedgraded condition
+        for status_cond in operator['status']['conditions']:
+            if status_cond['type'] ==  "Degraded":
+                
+                logging.info("Iteration %s: Operator %s: %s" % (iteration,operator['metadata']['name'], status_cond['status']))
```

Comment:
I think this line can be eliminated as it is sufficient to log aggregated cluster operator status.